src/appshell/qml/AboutMusicXMLDialog.qml

```diff
@@ -68,7 +68,7 @@ StyledDialogView {
                 StyledTextLabel {
                     width: parent.width
 
-                    text: qsTrc("appshell/about", "Copyright 2004 - 2017 the Contributors to the MusicXML Specification, published by the W3C Music Notation Community Group under the W3C Community Final Specification Agreement:")
+                    text: qsTrc("appshell/about", "Copyright 2004-2017 the Contributors to the MusicXML Specification, published by the W3C Music Notation Community Group under the W3C Community Final Specification Agreement:")
```

Comment:
I noticed another thing: currently, the musicxml website mentions copyright 2004-_2021_ (https://w3c.github.io/musicxml/).

Reply:
Thanks! Done.

Reply:
"Copyright © 2004-2021 the Contributors to the MusicXML Specification..." with ©

Reply:
Now it's like on this website.

Reply:
@cbjeukendrup Is it OK now or not?